draft-ietf-capport-architecture.xml

```diff
@@ -246,6 +246,8 @@
            avoid updating the default route to a device on captive WiFi network until
            network access restrictions have been lifted (excepting access to the User Portal) in
            the new network. This has been termed "make before break".</li>
+          <li>SHOULD support updates to the Captive Portal API URI from the network provisioning
+              service.</li>
         </ul>
```

Comment:
should this be grouped with the other SHOULD above?

Reply:
Good idea. Will do.